modules/core/src/main/scala/org/tessellation/package.scala

```diff
@@ -6,18 +6,18 @@ import cats.effect.Async
 
 import scala.util.control.NoStackTrace
 
-import org.tessellation.domain.aci.StateChannelOutput
 import org.tessellation.ext.kryo._
 import org.tessellation.schema.peer.PeerId
-import org.tessellation.security.SecurityProvider
+import org.tessellation.schema.security.SecurityProvider
+import org.tessellation.schema.statechannels.StateChannelOutput
 
 import eu.timepit.refined.auto._
-import eu.timepit.refined.numeric.Interval
+import eu.timepit.refined.numeric.Greater
 import io.estatico.newtype.ops._
 
 package object tessellation {
 
-  type CoreKryoRegistrationIdRange = Interval.Closed[700, 799]
+  type CoreKryoRegistrationIdRange = Greater[100]
```

Comment:
We don't want any constraint for the max registration number?

Reply:
we already discussed that and this is what @kpudlik did in his pull request.
if we're going to rediscuss it, let's create a separate task and handle/change all kryo id ranges

Reply:
Why `Greater[100]`? It will likely conflict with twitter chill registrations.

Reply:
after discussion, we've decided to keep it `Greater[300]`, because twitter/chill registers more than 100 classes